server/views/partials/components/sentence/template.njk

```diff
@@ -1,18 +1,11 @@
 {% from "hmpps/components/court-cases-release-dates/offence-card/macro.njk" import offenceCard %}
 
-{% if sentence.sentenceTypeDescription == 'Required' %}
-  {% set sentenceTypeDisplay = '<strong class="govuk-tag govuk-tag--blue">Required</strong>' %}
-{% else %}
-  {% set sentenceTypeDisplay = sentence.sentenceTypeDescription | default("N/A") %}
-{% endif %}
-{% set sentenceDate = sentence.sentenceDate | formatDate if sentence.sentenceDate else 'N/A' %}
-
 {# TODO sentence.consecutiveToChargeNumber is never set #}
 {{ offenceCard({
     offenceCode: sentence.offenceCode,
     offenceName: sentence.offenceDescription,
-    offenceStartDate: sentence.offenceStartDate | formatDate,
-    offenceEndDate: sentence.offenceEndDate | formatDate,
+    offenceStartDate: sentence.offenceStartDate | formatDate('DD/MM/YYYY'),
```

Comment:
we should probably have some test assertions somewhere that tests these dates are set correctly in the card. but can defer to another PR

probably the template.tests is the ideal place